client/snapper.cc

```diff
@@ -1203,11 +1212,16 @@ command_rollback(ProxySnappers* snappers, ProxySnapper* snapper)
     const string subvolume = config.getSubvolume();
     if (subvolume != "/")
     {
-        cerr << sformat(_("Command 'rollback' cannot be used on a non-root subvolume %s."),
+	cerr << sformat(_("Command 'rollback' cannot be used on a non-root subvolume %s."),
 			subvolume.c_str()) << endl;
-        exit(EXIT_FAILURE);
+	exit(EXIT_FAILURE);
     }
 
+    pair<bool, unsigned int> previous_default = filesystem->getDefault();
+
+    if (previous_default.first && scd1.description == "rollback backup")
+	scd1.description += sformat(" of #%d", previous_default.second);
```

Comment:
No, other descriptions also seem to be not translated, e.g. https://github.com/yast/yast-storage/blob/master/src/modules/StorageSnapper.rb#L82.